docs/4.0/utilities/colors.md

```diff
@@ -30,6 +34,15 @@ Similar to the contextual text color classes, easily set the background of an el
 <div class="p-3 mb-2 bg-white text-dark">.bg-white</div>
 {% endexample %}
 
+## Background gradient
+
+When `$enable-shadows` is set to true, you'll be able to use `.bg-gradient-` utility classes. **By default, `$enable-shadows` is disabled and the example below is intentionally broken.** This is done for easier customization from moment you start using Bootstrap. [Learn about our Sass options]({{ site.baseurl }}/docs/{{ site.docs_version }}/getting-started/theming/#sass-options) to enable these classes and more.
```

Comment:
You mean `$enable-gradients` instead of `shadows`, in this paragraph, right? 

My guess is that we'll end up replying a bunch of issues from users that skip reading (mea culpa), we'll see.